Automattic/vip
# Rename prepare-release.yml to npm-prepare-release.yml

## Description

This pull request will rename `.github/workflows/prepare-release.yml` to `.github/workflows/npm-prepare-release.yml` so the naming is consistent across repositories. There is no effect on functionality.


## Steps to Test

1. Merge PR.
2. Go to `Actions` and verify that `Prepare new release` action is active.
